### PR TITLE
Added `JsonMaid` and implementation

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #5 
Added `JsonMaid` and implementation that uses `com.fasterxml.jackson` to read/write json.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@g4s8 please, take a look

File: src/test/com/artipie/conda/meta/JsonMaidTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        final ByteArrayOutputStream stream = new ByteArrayOutputStream();
+        final JsonFactory factory = new JsonFactory();
+        final TestResource resource = new TestResource("repodata.json");
+        new JsonMaid.Jackson(
+            factory.createGenerator(stream, JsonEncoding.UTF8).useDefaultPrettyPrinter(),
+            factory.createParser(resource.asInputStream())
+        ).clean(Collections.emptySet());
+        MatcherAssert.assertThat(
+            stream.toByteArray(),
+            new IsEqual<>(resource.asBytes())
```
Comment (g4s8) [inline src/test/com/artipie/conda/meta/JsonMaidTest.java:34]: Please use the same charset for decoding as you use for encoding (UTF8 in this test), or just omit it for encoding to use system default (but same) for encoding and decoding

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova just one comment

**Comment by olenagerasimova** [comment]
@g4s8 thanks, corrected

**Pull request merged**